src/vue/forms/SubmitTradeOfferForm.vue

```diff
@@ -88,6 +103,37 @@
               </template>
             </template>
           </button>
+          <div class="app__button-tooltip">
+            <button
+              v-ripple
+              type="button"
+              @click="updateOrder"
+              class="app__form-submit-btn"
+              :disabled="!isEnoughOnBalance
+                || formMixin.isDisabled
+                || !isOwnerOrder"
+            >
+              <template v-if="!isShownToolTip">
+                {{ 'submit-trade-offers-form.update-order' | globalize }}
+              </template>
+              <template v-else-if="isOfferSubmitting">
+                {{ 'form-confirmation.submit-processing' | globalize }}
+              </template>
+              <template v-else>
+                {{ 'form-confirmation.button-text-ok' | globalize }}
+              </template>
```

Comment:
introduce a `<form-confirmation/>`. form-confirmation accepts `messageId`, so you can provide your `submit-trade-offers-form.update-order` to it instead of using your handmade confirmation. also please modify the message accordingly so it will look like "Your offer will be updated, please confirm". Or you can just use the default message and remove these at all